### PR TITLE
php: Support PHP 7, require at least 5.4

### LINKED ISSUE #2602: timeline to support PHP7
PHP 7 is around the corner and I tried the application with PHP7 and getting alot of deprecated issues with the Class methods having the same name as the class (the constructor) then nothing happens (a blank screen).

Is there a timeline when osTicket will be supporting PHP 7?

### LINKED ISSUE #2815: PHP 7 Compatibility
Hi,

When will osticket be compatible with PHP 7?

Or better question, is there a way to make the 1.10.rc.1 version work with PHP 7?

We upgraded our servers and osticket crashed. So bummed and really need to get it working again.

Thanks
Dan

### LINKED ISSUE #2819: v1.10-rc.2 install error with php7.0
cant install v1.10-rc.2 1st setup page giving a error 500 
php7 error log :
PHP Fatal error:  Cannot declare class Error, because the name is already in use in /***/include/class.error.php on line 0

install runs ok with PHP 5.6 but not with PHP 7.0

### PR DESCRIPTION
**Pull request by greezybacon**
This commit attempts to remove all coding standard warnings emitted by PHP 7.0. There's still some work to be done; but this gets things kinda working.

Fixes #2819
Fixes #2815
Fixes #2602

### CODE REVIEW

File: include/class.config.php [hunk 1]
```diff
@@ -127,7 +127,7 @@ function update($key, $value) {
             return false;
 
         $setting['value'] = $value;
-        $setting['updated'] = Misc::dbtime();
+        #$setting['updated'] = Misc::dbtime();
```
Comment (greezybacon) [inline include/class.config.php]: Oops

**Comment by greezybacon** [comment]
This gets much closer. The ORM stuff is fixed and the crazy function renames like `Ticket::create2` are now resolved. Now there are two remaining warnings emitted for some Form rendering methods. Almost there...

**Comment by 4nanook** [comment]
Is osTicket being actively developed anymore?  Still does not work under PHP7 and see no commits in six months.

**Comment by ntozier** [comment]
https://github.com/osTicket/osTicket/issues/3399

**Comment by 4nanook** [comment]
Much appreciate the update!

-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
  Eskimo North Linux Friendly Internet Access, Shell Accounts, and Hosting.
    Knowledgeable human assistance, not telephone trees or script readers.
  See our web site: http://www.eskimo.com/ (206) 812-0051 or (800) 246-6874.

**Pull request merged**